data/i18n/zh-cn/zh-cn.toml

```
@@ -28,11 +35,38 @@ other = "Twitter"
 [community_youtube_name]
 other = "YouTube"
 
+[cve_id]
+other = "CVE ID"
+
+[cve_issue_url]
+other = "CVE GitHub Issue URL"
```

https://kubernetes.io/zh-cn/docs/reference/issues-security/official-cve-feed/

```suggestion
other = "CVE GitHub Issue 網址"
```

---

URL is unlikely to be misunderstood, right?